manager/service.go

```diff
@@ -58,7 +58,7 @@ func New(qemuCfg qemu.Config, logger mglog.Logger, eventSvc events.Service, host
 }
 
 func (ms *managerService) Run(ctx context.Context, c *Computation) (string, error) {
-	ms.publishEvent("vm-provision", c.Id, "starting", json.RawMessage{})
+	ms.publishEvent("vm-provision", c.Id, c.Key, "starting", json.RawMessage{})
 	ac := agent.Computation{
```

Comment:
is key copied to ac before being added to agent, this is missing, also do manual tests

Reply:
The PR is still draft because it is pending manual tests